### PR TITLE
Feature/homework 1

### PR DESCRIPTION
**Pull request by franxx32**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function findArgsValueByKey(key, defaultValue) {
```
Comment (franxx32) [inline src/config.js:3]: Would be better to create defult value for defaultValue
`defaultValue = 'dev'`

File: src/logger.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = (msg) => {
```
Comment (franxx32) [inline src/logger.js]: Better to create logger object which will have different method for logs
For instance: logger.info(''), logger.error('') and so on

Comment (korobko-virtru) [reply src/logger.js]: Fixed

File: src/config.js (hunk 1 above)
Comment (korobko-virtru) [reply src/config.js:3]: Fixed

**Pull request merged**